lib/templates/settingsConsoleModuleSelector.mustache

```diff
@@ -4,8 +4,12 @@
             <h3 class="categoryButton" data-category="{{name}}">{{name}}</h3>
             <ul>
 				{{#modules}}
-                    <li class="moduleButton {{#isEnabled}}enabled{{/isEnabled}}" data-module="{{moduleID}}">
+                    <li class="moduleButton {{#isEnabled}}enabled{{/isEnabled}}" data-module="{{moduleID}}" title="{{description}}">
 						{{moduleName}}
+						<br>
```

Comment:
Instead of a `<br>`, how about adding `.moduleButton small { display: block }` to the settings console CSS?